src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1490,6 +1490,37 @@ internal static bool TryGetSpecialTypeMember<TSymbol>(CSharpCompilation compilat
             return true;
         }
 
+        /// <summary>
+        /// This is a layer on top of the Compilation version that generates a diagnostic if the special
+        /// member isn't found.
+        /// </summary>
+        internal Symbol GetWellKnownTypeMember(WellKnownMember member, DiagnosticBag diagnostics, SyntaxNode syntax)
```

Comment:
Tempted to call this `TryGetWellKnownTypeMember` too.

Reply:
This is for consistency with the already existing `GetSpecialTypeMember` and `TryGetSpecialTypeMember`